### ISSUE TITLE
Can not run

### ISSUE THREAD

**Issue by DrRingo** [crdoconnor/orji#1]
I run the example in your README and get this error: TypeError: cannot unpack non-iterable NoneType object

**Comment by crdoconnor** [comment]
can you upload the exact files you used? i cant seem to reproduce the issue.

**Comment by crdoconnor** [comment]
There might be something slightly different about the whitespace in your
files.

**Comment by DrRingo** [comment]
Here you are. I don't know what specific white space I use. I just copy your example to vscode on Windows 11, save the file, and run on WSL
![Screenshot 2024-03-03 152324](https://github.com/crdoconnor/orji/assets/13098629/eeb93324-8960-4304-b392-f7df98e5f1f0)

**Comment by DrRingo** [comment]
Your "latexed" function may not convert orgmode format to latex. 
Example: I write bold, italic in orgmode note, but Orji not auto convert it to latex format, even when I use "latexed" after "body"
Please fix this, your project have inspired me a lot. I use both orgmode and latex heavily in my daily job

**Comment by crdoconnor** [comment]
I can't verify why exactly you were getting the above error for the default examples, but I've been getting it too for files without any actual notes like this:

```
This is the preface.\n\n
```

I also tested it with windows line endings and it seemed to work.

It's likely you didn't copy the whole file or copied and pasted without the bullet points '*' or something.

**Comment by crdoconnor** [comment]
> Your "latexed" function may not convert orgmode format to latex.

Yep, I'm afraid I did a very minimal version that worked for my purposes. I only cared about ampersands and URLs.

I've raised this as a separate issue here: https://github.com/crdoconnor/orji/issues/2

**Comment by crdoconnor** [comment]
This issue should be fixed on the main branch of orgmunge. It's not yet released yet though.
